### PR TITLE
Rename Pumpkin Pad to Pumkinpad

### PR DESCRIPTION
**Pull request by zykrah**
## Description

As in the title, requested by Laser Ninja themselves.

image

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by fauxpark — CHANGES_REQUESTED** [review]
Alias will need to be added to `data/mappings/keyboard_aliases.hjson`.

**Comment by zykrah** [comment]
> Alias will need to be added to `data/mappings/keyboard_aliases.hjson`.

What kind of alias? An alias to link `laser_ninja/pumkinpad` to `pumkinpad` or to link the old name: `laser_ninja/pumkinpad` to `laser_ninja/pumkin_pad`?

**Comment by fauxpark** [comment]
The old FQN to the new one. Have a look at that file, you should see what I mean.

File: data/mappings/keyboard_aliases.hjson [hunk 1]
```diff
@@ -927,6 +927,9 @@
     "kprepublic/bm68hsrgb": {
         "target": "kprepublic/bm68hsrgb/rev1"
     },
+    "laser_ninja/pumpkin_pad": {
+        "target": "laser_ninja/pumpkinpad"
+    },
```
Comment (fauxpark) [inline data/mappings/keyboard_aliases.hjson:932]: Needs to be further up, above `lattice60`. This section is for boards moved in 2022Q1.

**Review by fauxpark — APPROVED** [review]

**Review by zvecr — APPROVED** [review]

**Pull request merged**